package.json

```
@@ -40,19 +40,23 @@
 		"geth:start": "WEB3_SYSTEM_TEST_BACKEND=geth && ./scripts/geth.sh start",
 		"geth:start:background": "WEB3_SYSTEM_TEST_BACKEND=geth && ./scripts/geth.sh start 1",
 		"geth:stop": "./scripts/geth.sh stop",
+		"ipc:start:background": "WEB3_SYSTEM_TEST_BACKEND=ipc && ./scripts/ipc.sh start 1",
+		"ipc:stop": "./scripts/ipc.sh stop",
 		"lint": "lerna run lint --stream --parallel",
 		"lint:fix": "lerna run lint:fix --stream --parallel",
 		"format": "lerna run format --stream --parallel",
 		"test": "lerna run test --stream --parallel",
 		"test:unit": "lerna run test:unit --stream --parallel",
 		"test:integration": "lerna run test:integration --stream",
+		"test:integration:ipc": "cd ./packages/web3 && yarn test:integration test/integration/ipc.test.ts",
```

Would suggest to keep it consistent with other scripts and create that script only in the `web3` package. 


```suggestion
		"test:integration:ipc": "lerna run test:integration:ipc --stream",
```